Karma/Migrations/KarmaContextModelSnapshot.cs

```diff
@@ -202,34 +202,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("Post");
                 });
 
-            modelBuilder.Entity("Karma.Models.Report", b =>
-                {
-                    b.Property<int>("Id")
-                        .ValueGeneratedOnAdd()
-                        .HasColumnType("INTEGER");
-
-                    b.Property<int>("PostId")
-                        .HasColumnType("INTEGER");
-
-                    b.Property<string>("PostOwnerId")
-                        .IsRequired()
-                        .HasColumnType("TEXT");
-
-                    b.Property<string>("ReportMessage")
-                        .HasColumnType("TEXT");
-
-                    b.Property<int>("ReportState")
-                        .HasColumnType("INTEGER");
-
-                    b.Property<string>("ReporterId")
-                        .IsRequired()
-                        .HasColumnType("TEXT");
-
-                    b.HasKey("Id");
-
-                    b.ToTable("Report");
-                });
-
```

Comment:
Are reports no longer needed? Or why are we removing them from database file?

Reply:
Good point, happened during rebase. Going to fix that

Reply:
Fixed. It should be fine now. I also added AllowAnonymous attributes because I couldn't access pages without it.